Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:alpine
+
+WORKDIR /app
+
+COPY package.json yarn.lock ./
+
+RUN yarn install
+
+COPY ./src ./src
+COPY . ./
```

Comment:
Doesn't this also copy `src`?

Reply:
you're right, doesn't really help caching here